### PR TITLE
1 fix herocard component style

### PR DESCRIPTION
**Pull request by Byte-Block**
Most of the style is fixed.

### CODE REVIEW

**Comment by Byte-Block** [comment]
Close #1

File: src/components/HeroCard/HeroCard.css [hunk 1]
```diff
@@ -11,7 +12,7 @@
 .HeroCard__Card-Image {
   display: block;
   margin: auto;
-  height: 200px;
+  height: 300px;
```
Comment (veljkobugaric) [inline src/components/HeroCard/HeroCard.css:15]: Style approved 👍

**Review by veljkobugaric — COMMENTED** [review]
Review approved 👍

**Review by Byte-Block — COMMENTED** [review]
🚢 🔱

**Review by veljkobugaric — APPROVED** [review]
🚢 🔱

**Pull request merged**